src/main/java/com/techsophy/tsf/wrapperservice/service/impl/ProcessServiceImpl.java

```diff
@@ -413,6 +411,50 @@ public void completeTaskWithChecklistItemId(ChecklistItemInstanceDTO checklistIt
 
     }
 
+    @Override
+    public void deleteProcessById(String ticketNumber,String id) throws JsonProcessingException
+    {
+        Map<String,String> ticketData =new HashMap<>();
+        String ticketNumberValue =REVIEW_TICKET+ticketNumber;
+        ticketData.put(NAME, ticketNumberValue);
+        String ticketDetailsUrl=gatewayURI+camundaServletContextPath+ENGINE_REST+TASK;
+        UriComponentsBuilder uriComponentsBuilder1 = UriComponentsBuilder.fromHttpUrl(ticketDetailsUrl);
+        HttpHeaders httpHeaders1 = new HttpHeaders();
+        httpHeaders1.add(HttpHeaders.AUTHORIZATION, getBearerToken());
+        httpHeaders1.setContentType(MediaType.APPLICATION_JSON);
+        HttpEntity<?> httpEntity1 = new HttpEntity<Object>(ticketData,httpHeaders1);
+        ResponseEntity responseEntity1 = restTemplate.exchange(uriComponentsBuilder1.toUriString(), HttpMethod.POST, httpEntity1, Object.class);
+        List<Map<String,Object>> responseEntity1Body = (List<Map<String, Object>>) responseEntity1.getBody();
+        String processInstanceId= String.valueOf(responseEntity1Body.get(0).get(PROCESS_INSTANCE_ID));
+        String deleteTicketUrl=gatewayURI+camundaServletContextPath+ENGINE_REST+DELETE_TASK_BY_PROCESS_INSTANCE_ID+processInstanceId;
+        UriComponentsBuilder uriComponentsBuilder2 = UriComponentsBuilder.fromHttpUrl(deleteTicketUrl);
+        HttpHeaders deletehttpHeaders = new HttpHeaders();
+        deletehttpHeaders.add(HttpHeaders.AUTHORIZATION, getBearerToken());
+        deletehttpHeaders.setContentType(MediaType.APPLICATION_JSON);
+        HttpEntity<?> deletehttpEntity = new HttpEntity<Object>(null, deletehttpHeaders);
+        ResponseEntity<Object> response = restTemplate.exchange(uriComponentsBuilder2.toUriString(), HttpMethod.DELETE, deletehttpEntity, Object.class);
+
+        if(response.getStatusCode().is2xxSuccessful())
+        {
+            StatusCodeUpdateDTO statusCodeUpdateDTO = new StatusCodeUpdateDTO();
+            statusCodeUpdateDTO.setFormId("951380543294529536");
```

Comment:
Why hardcoded id?

Reply:
Here, we are using same form for storing the data each time.